certora/specs/Badger-datapunk-2.spec

```diff
@@ -0,0 +1,49 @@
+// Check Points - This is really long running
+
+import "erc20.spec"
+import "rewardsManagerMethods.spec"
```

Comment:
we cannot run your spec with this import. I didn't find it in your repo either. If this import is not used and we can remove it, let me know asap